README.md

```diff
@@ -176,15 +176,39 @@ sgnops init-candidate --commission-rate 1 --min-self-stake 1 --rate-lock-end-tim
 
 It will take a while to complete the transactions on Ropsten.
 
-13. (Optional) In another terminal window, start an SGN gateway server:
+14. Delegate 10000 Ropsten CELR to yourself
```

Comment:
10000 is a recommendation, right? Make a note on what is the required CELR

Reply:
Done.